.prettierignore

```diff
@@ -141,3 +141,6 @@ packages/suite-data/files/translations/master.json
 packages/suite-data/files/videos/lottie/*.json
 
 /.nx/cache
+
+# NAPI generated files 
+packages/transport-bluetooth/napi
```

Comment:
it this intended? 
if yes then it's not working because files are included anyway, but i think we want to track those builds, no?

or if we dont want to track it then maybe
```
yarn global add @napi-rs/cli && yarn build:napi
```

should be done not in the Dockerfile but as a part of transport-bluetooth build:lib ?

Reply:
ah, its priettier not git :)